dart/lib/src/sentry_client.dart

```diff
@@ -485,6 +485,75 @@ class SentryClient {
     );
   }
 
+  @internal
+  Future<void> captureLog(
+    SentryLog log, {
+    Scope? scope,
+  }) async {
+    if (!_options.enableLogs) {
+      return;
+    }
+
+    log.attributes['sentry.sdk.name'] = SentryLogAttribute.string(
+      _options.sdk.name,
+    );
+    log.attributes['sentry.sdk.version'] = SentryLogAttribute.string(
+      _options.sdk.version,
+    );
+    final environment = _options.environment;
+    if (environment != null) {
+      log.attributes['sentry.environment'] = SentryLogAttribute.string(
+        environment,
+      );
+    }
+    final release = _options.release;
+    if (release != null) {
+      log.attributes['sentry.release'] = SentryLogAttribute.string(
+        release,
+      );
+    }
+    final span = scope?.span;
+    final propagationContext = scope?.propagationContext;
+    if (span != null) {
+      log.attributes['sentry.trace.parent_span_id'] = SentryLogAttribute.string(
+        span.context.spanId.toString(),
+      );
+      log.traceId = span.context.traceId;
+    } else if (propagationContext != null) {
+      log.traceId = propagationContext.traceId;
```

Comment:
when we start a transaction aren't we setting the transaction trace I'd to the propagationacontext traceId meaning we can just always use the propagation trace id?

please checj

Reply:
https://github.com/getsentry/sentry-dart/blob/bc3ad3c691cee925a540e972eb893a958ebf53d2/dart/lib/src/hub.dart#L469

Yeah, every transaction that is started gets the current propagation context, so it probably make sense to just use it.

Reply:
we could still add the spanId of the span but always use the propagation context trace Id